.travis.yml

```diff
@@ -8,7 +8,7 @@ language: go
 go_import_path: github.com/HewlettPackard/terraform-provider-oneview
 
 go:
-    - 1.8
+    - 1.11.3
```

Comment:
Lets keep it as 1.11 aligned with Oneview-go

Reply:
Updated the go version